companies/1and1-gb.json

```diff
@@ -1,16 +1,16 @@
 {
     "slug": "1and1-gb",
     "relevant-countries": [
-        "gb"
+        "gs"
     ],
     "categories": [
         "telecommunication",
         "commerce"
     ],
     "name": "1&1 Internet Ltd.",
-    "address": "Discovery House\n154 Southgate Street\nGloucester\nGL1 2EX\nUnited Kingdom",
+    "address": "1&1 Internet Ltd.\nDiscovery House\n154 Southgate Street\nGloucester\nGL1 2EX\nUnited Kingdom",
     "phone": "+44 333 336 5691",
-    "email": "privacypolicy@1and1.co.uk",
+    "email": "privacypolicy@1and1.co@uk",
```

Comment:
🚫 **[recordlint]** <[pass-schema](https://github.com/datenanfragen/data#data-formats)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record did not pass schema validation: “The value at /email must be a valid idn-email string but it was not.”

Raw error:
```json
{
    "path": "$.email",
    "pointer": "/email",
    "message": "The value at /email must be a valid idn-email string but it was not.",
    "data": "privacypolicy@1and1.co@uk"
}
```